lib/capistrano/data_migrate.rb

```diff
@@ -0,0 +1,3 @@
+require 'capistrano'
```

Comment:
capistrano is not a dependency of the gem, so this probably needs some checks/try-catch and probably documentation